src/backend/columnar/cstore_tableam.c

```diff
@@ -1143,60 +1127,12 @@ CStoreSubXactCallback(SubXactEvent event, SubTransactionId mySubid,
 }
 
 
-#if PG_VERSION_NUM >= 130000
-static void
-CStoreTableAMProcessUtility(PlannedStmt *plannedStatement,
-							const char *queryString,
-							ProcessUtilityContext context,
-							ParamListInfo paramListInfo,
-							QueryEnvironment *queryEnvironment,
-							DestReceiver *destReceiver,
-							QueryCompletion *queryCompletion)
-#else
-static void
-CStoreTableAMProcessUtility(PlannedStmt * plannedStatement,
-							const char * queryString,
-							ProcessUtilityContext context,
-							ParamListInfo paramListInfo,
-							QueryEnvironment * queryEnvironment,
-							DestReceiver * destReceiver,
-							char * completionTag)
-#endif
-{
-	Node *parseTree = plannedStatement->utilityStmt;
-
-	if (nodeTag(parseTree) == T_CreateTrigStmt)
-	{
-		CreateTrigStmt *createTrigStmt = (CreateTrigStmt *) parseTree;
-
-		Relation rel = relation_openrv(createTrigStmt->relation, AccessShareLock);
-		bool isCStore = rel->rd_tableam == GetColumnarTableAmRoutine();
-		relation_close(rel, AccessShareLock);
-
-		if (isCStore &&
-			createTrigStmt->row &&
-			createTrigStmt->timing == TRIGGER_TYPE_AFTER)
-		{
-			ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-							errmsg(
-								"AFTER ROW triggers are not supported for columnstore access method"),
-							errhint("Consider an AFTER STATEMENT trigger instead.")));
-		}
-	}
-
-	CALL_PREVIOUS_UTILITY();
```

Comment:
definition of `CALL_PREVIOUS_UTILITY` can be removed as it is not used anymore

Reply:
Fixed.